app/views/communicart_mailer/_comments.html.haml

```diff
@@ -0,0 +1,21 @@
+- if comments.any?
+  %table.submodel-section
+    %tr
+      %td.w-container{:colspan => "2"}
+        %strong Comments
+    - comments.each do |c|
+      %tr
+      - if c.update_comment
+        class="comment-update"
+      \&gt;
```

Comment:
Regarding the text only emails... I see what you're getting at. Yes, haml is pointless if there is no markup. We should leave erb as @cmc333333  suggested.